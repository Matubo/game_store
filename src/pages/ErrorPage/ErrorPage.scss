.error-page {
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
  flex-wrap: wrap;
  max-width: 100%;
}
.error-page__img {
  max-width: 100%;
}
.error-page__text {
  font-size: xx-large;
  font-weight: bolder;
}
