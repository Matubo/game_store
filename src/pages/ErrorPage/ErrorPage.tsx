export default function ErrorPage() {
  return <div>ErrorPage mather fucker</div>;
}
