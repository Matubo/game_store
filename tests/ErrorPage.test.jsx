import { render, screen } from '@testing-library/react';
import ErrorPage from 'src/pages/ErrorPage/ErrorPage';

describe('ErrorPage', () => {
  const { asFragment } = render(<ErrorPage></ErrorPage>);
  test('snapTest', async () => {
    const errorMessage = screen.getByText("404 This is not the page you're looking for");
    expect(errorMessage).toBeInTheDocument();
  });
  test('have error img', async () => {
    render(<ErrorPage></ErrorPage>);
    const errorImg = screen.getByAltText('404');
    expect(errorImg).toBeInTheDocument();
  });
  test('snap test', async () => {
    const fragment = asFragment();
    expect(fragment).toMatchSnapshot();
  });
});
