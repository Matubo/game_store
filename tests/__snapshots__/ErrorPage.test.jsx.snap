// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage snap test 1`] = `<DocumentFragment />`;
